pygments/lexers/shell.py

```diff
@@ -847,3 +848,59 @@ class FishShellLexer(RegexLexer):
             include('root'),
         ],
     }
+
+class ExeclineLexer(RegexLexer):
+    """
+    Lexer for Laurent Bercot's execline language
+    """
```

Comment:
Needs a `versionadded` statement, this will go into 2.6.

Reply:
Ah, it looks like I missed the 2.6 release; I'll bump this to 2.7 instead.